content/docs/azure/azureEndtoEnd.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "End-to-End Pipeline Deployment on Azure"
+description = "An end-to-end guide to using the azurepipeline example"
```

Comment:
Readers won't know what "the azurepipeline example" means, and there's no link (because it's a subtitle) to help. Rather, describe what the pipeline does.